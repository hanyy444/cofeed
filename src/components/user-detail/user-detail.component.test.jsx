import {render, screen} from '@testing-library/react'
import UserDetail from './user-detail.component'

describe('Test user-detail component', () => {
    it('Should render without crashing', () => {
        expect.assertions(2)
        render(<UserDetail icon={'test'} text={'test'} />)
        const userDetails = screen.getByTestId('user-detail')
        expect(userDetails).toBeInTheDocument()
        expect(userDetails).toMatchSnapshot()
    })
})