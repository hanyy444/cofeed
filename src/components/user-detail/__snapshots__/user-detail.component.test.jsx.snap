// Vitest Snapshot v1

exports[`Test user-detail component > Should render without crashing 1`] = `
<div
  class="user-detail"
  data-testid="user-detail"
  id="user-detail"
>
  test
  <p>
     
    test
     
  </p>
</div>
`;
